lib/slackbot.rb

```diff
@@ -0,0 +1,28 @@
+require 'slack-ruby-bot'
+require_relative './darksky.rb'
+require_relative './google_coordinates.rb'
+
+class Slackbot < SlackRubyBot::Bot
+  command 'weather now' do |client, data, match|
+    location = match[:expression] || 'Washington DC'
+    location_description, coordinates = GoogleCoordinates.location_info(location)
+    if coordinates.blank?
+      client.say(text: location_description, channel: data.channel)
+    else
+      weather_now = Darksky.weather_now(coordinates)
+      client.say(text: "Weather now for #{location_description}: #{weather_now}", channel: data.channel)
+    end
+  end
+
+  command 'weather tomorrow' do |client, data, match|
+    #todo: DRY these functions up
```

Comment:
You aptly state that these two `command`s could be DRY'd up. It seems like these are almost entirely the same. REQUEST: Would it be possible to take a quick stab at DRY-ing them?